packages/sol6/src/yang/etsi-nfv-common.yang

```diff
@@ -282,7 +282,7 @@ module etsi-nfv-common {
     leaf root {
       type uint64;
       units "Mbps";
-      mandatory true;
+      default 0;
```

Comment:
Thinking about the implication of this change, having it as mandatory meant that the service would not configure unless it was specified. Changing it to a default value of 0 means, the service will get configured, but it will not work, as who can have a root with 0 throughput. That means someone has to realize, and change it to a non-zero value. Is that the behavior we want?

Reply:
I agree that is the draw-back with the defaults. Our NFVO does not use the values, so for 'us' this is absolutely ok. I would prefer it to not be mandatory of course, but that would go against the IFA I guess?

Reply:
Yes, as specified, IFA011 makes it mandatory. But, I am curious. If you are instantiating a VL, why would you not want to specify (a non-zero) value for root and/or leaf? What is the system supposed to assume for maximum and minimum bitrate?